mkt/reviewers/views.py

```diff
@@ -448,17 +444,12 @@ def app_review(request, addon):
         # them to index the app or call update_version() if that wasn't done
         # before already.
         if request.method == 'POST':
-            try:
+            with transaction.atomic():
```

Comment:
That's not 1105507, no. Without moving anything to celery, this bit of code is obsolete and could probably be replaced by something doing signals normally in the code being executed before.